### PR TITLE
Feature/local go coverage/sterling

### PR DESCRIPTION
**Pull request by sdreyer**
Runs coverage testing locally and opens in browser

### CODE REVIEW

File: .github/workflows/link-check.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ name: Check All Public Links
 on:
   push:
     branches:
-      - main
+      - client
```
Comment (sdreyer) [inline .github/workflows/link-check.yml:6]: Mistake, will fix

File: ff/serving/check_coverage.sh [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+go test -coverpkg=./... -coverprofile cover.out ./...
+
+# Convert Coverage To HTML
+go tool cover -html=cover.out -o coverage.html
+
+# Cleanup Files
+rm cover.out
+
+# Open In Browser
+open coverage.html
```
Comment (simba-git) [inline ff/serving/check_coverage.sh:15]: This may be mac specific

File: ff/serving/check_coverage.sh [hunk 3]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```
Comment (simba-git) [inline ff/serving/check_coverage.sh:1]: Unrelated to this file, but add .idea to .gitignore

File: ff/serving/check_coverage.sh (hunk 2 above)
Comment (sdreyer) [reply ff/serving/check_coverage.sh:15]: Should I submit this first so saadhvi can use it then add support for windows and linux? Or should that all go in one go?

Comment (sdreyer) [reply ff/serving/check_coverage.sh]: Changed To Be Mac Specific

File: ff/serving/check_coverage.sh (hunk 3 above)
Comment (sdreyer) [reply ff/serving/check_coverage.sh]: Added

**Pull request merged**